apps/samradsgatt/components/CaseDetails/LeftSideColumn/CaseBreadcrumbs.tsx

```diff
@@ -1,13 +1,18 @@
 import { Box, Breadcrumbs } from '@island.is/island-ui/core'
+import { useLocation } from 'react-use'
 
-const CaseBreadcrumbs = () => {
+interface CaseBreadcrumbsProps {
+  caseNumber: string
+}
+
```

Comment:
spurning hvort breadcrumbs ætti ekki að vera heimskur kompónent þannig það sé hægt að nýta hann all staðar.
Þetta er óskýrt í hönnuninni en ætti að vera í boði að hafa 2 hlekki í breadcrumbs (td. Samráðsgátt - Tölfræði) eða 3 hlekki (Samráðsgátt - Öll mál - Mál nr. X)

Reply:
Í staðinn fyrir að vera með sér component CaseBreadCrumbs þá setti ég bara Breadcrumbs componentinn beint inn eins og er yfirleitt gert hjá island.is